content/en/network_monitoring/devices/config_management.md

```diff
@@ -0,0 +1,186 @@
+        profile: <PROFILE_NAME>
+        ## @param auth - object - required
+        ## Authentication credentials to connect to the network device.
+        auth:
+          ## @param username - string - required
+          ## Username to authenticate to the network device.
+          username: <USERNAME>
+          ## @param password - string - required (if private_key_file is not provided)
+          ## Password to authenticate to the network device.
+          ## Used as a fallback after private key authentication if both are provided.
+          password: <PASSWORD>
+          ## @param private_key_file - string - optional
+          ## Path to the SSH private key file for authentication.
+          ## At least one of password or private_key_file must be provided.
+          private_key_file: /path/to/private_key
+   ```
+
+2. Optionally, if your devices require specific SSH algorithms, use the following configuration:
+
+   ```yaml
+   init_config:
+     ## @param ciphers - list of strings - optional
+     ## List of SSH encryption ciphers to use for the connection.
+     ## If not specified, the SSH library will use its default ciphers.
+     ciphers: [aes128-gcm@openssh.com, aes128-ctr, aes192-ctr]
+     key_exchanges: [diffie-hellman-group14-sha256, ecdh-sha2-nistp256]
+     host_key_algorithms: [ssh-ed25519]
+   ```
+
+   See the full [configuration file][7] for more details.
+
+3. Configure the device profile if not already specified in your `conf.yaml`. Default profiles are located at `/conf.d/network_config_management.d/default_profiles`. See [device profiles](#device-profiles) for more information.
+
+   **Note**: NCM uses dedicated default profiles that differ from SNMP device profiles. Custom profiles are not supported.
+
+4. Restart the Agent to apply the configuration changes.
+
+## Viewing configurations
+
+Configuration Management is accessible from the device side panel in Network Device Monitoring:
+
+1. Navigate to [Network Device Monitoring][3].
+2. Select a device from the device list or from any NDM visualization such as [Device Geomap][4] or the [Device Topology][5] map.
+3. Open the **Configuration** tab in the device side panel.
+
+   {{< img src="/network_device_monitoring/config_mgmt/config_tab_redacted.png" alt="Network Device Management side panel, highlighting the Configuration tab." style="width:100%;" >}}
+
+   On the Configuration tab, you can filter the configuration list to display:
+   - **All**: Shows both running and startup configurations
+   - **Running**: The active, live configuration running on the device
+   - **Startup**: The saved configuration that loads when the device boots
+
+   **Running configuration**
+   : The active configuration with all live changes applied to the device. This is what the device uses in real-time.
+
+   **Startup configuration**
+   : The saved configuration that persists across reboots. When a device restarts, it loads this configuration.
```

Comment:
This is pretty repetitive of the list above. Consider consolidating into one list?